product-service/src/getAllProducts.ts

```diff
@@ -0,0 +1,14 @@
+import db from './db';
+import 'source-map-support/register';
+
+export const getAllProducts = async (_event, _context) => {
```

Comment:
Please add types

Reply:
Added return types for the functions (`APIGatewayProxyHandler` from `aws-lambda` library)